backend/src/models/UsersManager.js

```diff
@@ -1,36 +1,44 @@
 const AbstractManager = require("./AbstractManager");
+const bcrypt = require("bcrypt");
+const saltRounds = 10;
+const myPlaintextPassword = "s0//P4$$w0rD";
+const someOtherPlaintextPassword = "not_bacon";
```

Comment:
These variables are useless, aren't they?

Reply:
More, they are the example variables from the bcrypt package and are not used in this file